tempfork/gliderlabs/ssh/context.go

```diff
@@ -55,8 +55,6 @@ var (
 	// ContextKeyPublicKey is a context key for use with Contexts in this package.
 	// The associated value will be of type PublicKey.
 	ContextKeyPublicKey = &contextKey{"public-key"}
-
-	ContextKeySendAuthBanner = &contextKey{"send-auth-banner"}
```

Comment:
We don't need gliderlabs to support auth banners anymore, as we now use crypto/ssh's authentication callbacks directly.